docs/run_test/run.md

```diff
@@ -0,0 +1,24 @@
+# Run LISA
+
+- ## [Quick start](quick_run.md)
+  - ### [Getting started with Azure](quick_run.md)
+- ## [Test classification](#test-classification)
+  - ### [Microsoft tests](microsoft_tests.md)
+    - [How to run](microsoft_tests.md#how-to)
+    - [Test priority](microsoft_tests.md#test-priority)
+    - [Test tier](microsoft_tests.md#test-tier)
+    - [Test cases specification](microsoft_tests.md#test-cases-specification)
+  - ### [LISAv2 legacy tests](run_legacy.md)
+    - [Preparation](run_legacy.md#preparation)
+    - [Limitations](run_legacy.md#limitation)
+    - [Roadmap](run_legacy.md#roadmap)
+  - ### [Table of test cases]()
+- ## [Platform](platform.md)
+  - ### [Azure](platform.md#run-on-azure)
+  - ### [Ready platform](platform.md#run-on-ready-computers)
+- ## [Runbook](runbook.md)
+  - ### [How to configure](runbook.md#how-to)
+  - ### [Customization](runbook.md#reference)
+- ## [Test results](understand_results.md)
```

Comment:
The name is better to task oriented. How about change "Test results" to "Analyze test results"?